### PR TITLE
CHE-1326: avoid creation recipe on workspace creation from stack and overriding environment

### PR DESCRIPTION
**Pull request by ashumilova**
Signed-off-by: Ann Shumilova ashumilova@codenvy.com

Partly depends on https://jira.codenvycorp.com/browse/CODENVY-651

### CODE REVIEW

File: dashboard/src/app/projects/create-project/create-project.controller.js [hunk 1]
```diff
@@ -831,34 +831,17 @@ export class CreateProjectCtrl {
     }
     // check workspace is selected
     if (option === 'create-workspace') {
+      let source = {};
```
Comment (benoitf) [inline dashboard/src/app/projects/create-project/create-project.controller.js]: is that source should be only in the else block ?

Comment (ashumilova) [inline dashboard/src/app/projects/create-project/create-project.controller.js]: yes, will move it there

**Comment by ashumilova** [comment]
@benoitf @evoevodin @AndrienkoAleksandr  please check

**Comment by voievodin** [comment]
serverside is ok

**Comment by AndrienkoAleksandr** [comment]
LGTM

**Pull request merged**